<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadItem</class>
 <widget class="QWidget" name="DownloadItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>206</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="m_lblFileIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="m_progressDownload">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="SqueezeLabel" name="m_lblRemoteFilename" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" stdset="0">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_lblLocalFilename">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SqueezeLabel" name="m_lblInfoDownload" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnTryAgain">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Try again</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnStopDownload">
     <property name="text">
      <string>&amp;Stop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnOpenFile">
     <property name="text">
      <string>&amp;Open file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnOpenFolder">
     <property name="text">
      <string>Open &amp;directory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqueezeLabel</class>
   <extends>QWidget</extends>
   <header>squeezelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
